packages/turf-rhumb-bearing/index.js

```diff
@@ -28,16 +27,69 @@ export default function (start, end, final) {
     if (!start) throw new Error('start point is required');
     if (!end) throw new Error('end point is required');
 
-    var coordsStart = getCoord(start);
-    var coordsEnd = getCoord(end);
-    var origin = new LatLonSpherical(coordsStart[1], coordsStart[0]);
-    var destination = new LatLonSpherical(coordsEnd[1], coordsEnd[0]);
-    var bear360;
+    const coordsStart = getCoord(start);
+    const coordsEnd = getCoord(end);
+    const origin = new LatLon(coordsStart[1], coordsStart[0]);
+    const destination = new LatLon(coordsEnd[1], coordsEnd[0]);
+    let bear360;
 
     if (final) bear360 = destination.rhumbBearingTo(origin);
     else bear360 = origin.rhumbBearingTo(destination);
 
-    var bear180 = (bear360 > 180) ? -(360 - bear360) : bear360;
+    const bear180 = (bear360 > 180) ? -(360 - bear360) : bear360;
 
     return bear180;
 }
+
+
+// The following functions have been taken from Geodesy
+// (https://github.com/chrisveness/geodesy/blob/master/latlon-spherical.js)
+// in order to remove it as dpendency and reduce the final bundle size
+
+/**
+ * Creates a LatLon point on the earth's surface at the specified latitude / longitude.
+ *
+ * @private
+ * @constructor
+ * @param {number} lat - Latitude in degrees.
+ * @param {number} lon - Longitude in degrees.
+ *
+ * @example
+ *     var p1 = new LatLon(52.205, 0.119);
+ */
+function LatLon(lat, lon) {
```

Comment:
🤔 I wonder if we could remove the Class Object entirely and simply just make a `rhumbBearingTo` method?